### PR TITLE
Adding documentation for data.connection.setHeader

### PR DESCRIPTION
**Pull request by gcoonrod**
Doc update for https://github.com/actionhero/actionhero/pull/1053

### CODE REVIEW

**Comment by gcoonrod** [comment]
Need to extend the update to include information about the generic use case.

File: pages/docs/core/actions.js [hunk 1]
```diff
@@ -416,6 +416,7 @@ export default class extends DocsPageWithNav {
...
                 <p>You can also modify properties of the connection by accessing <code>data.connection</code>, IE changing the response header for a HTTP request.</p>
                 <p>If you don't want your action to respond to the client, or you have already sent data to the client (perhaps you already rendered a file to them or sent an error HTTP header), you can set <code>data.toRender = false;</code></p>
+                <p>If you are certain that your action is only going to be handled by a web server, then a convience function has been provided to you via <code>data.connection.setHeader()</code>. This function is a proxy to the <a href='https://nodejs.org/api/http.html#http_response_setheader_name_value'>Node HTTP Response setHeader</a> function and allows you to set response headers without having to drill into the <code>data.connection.rawConnection</code> object. Please be aware, the <code>data.connection.setHeader()</code> function will only be available if your action is being handled by a web server. Other server types will throw an exception.</p>
```
Comment (evantahler) [inline pages/docs/core/actions.js]: ... to learn more about how to set custom connection options, click here.

**Comment by evantahler** [comment]
One note, otherwise great!

**Comment by gcoonrod** [comment]
@evantahler What note?

**Pull request merged**